bodhi/server/scripts/approve_testing.py

```diff
@@ -51,6 +52,20 @@ def usage(argv):
     sys.exit(1)
 
 
+def clear_side_aux_tags(client_session, update):
+    for tag in (
+        update.from_tag + '-pending-signing',
```

Comment:
I made them configurable in #3481, you can call it like this `update.release.get_pending_signing_side_tag(update.from_tag)`.
Similar for testing side tag `update.release.get_testing_side_tag(update.from_tag)`.

Configuration is done as for any release specific config:
`rawhide.koji-testing-side-tag`
`rawhide.koji-pending-signing-side-tag`